cmd/talisman/talisman.go

```diff
@@ -133,4 +135,5 @@ func init() {
 		"This means all the data will be wiped off from the cluster and cannot be recovered")
 	flag.StringVar(&wiperImage, "wiperimage", "", "Node wiper image to use for the upgrade")
 	flag.StringVar(&wiperTag, "wipertag", "", "Node wiper tag to use for the upgrade")
+	flag.IntVar(&timeoutPerNode, "timeoutpernode", 600, "Timeout per node in seconds for performing upgrade")
```

Comment:
rename command line flag and global variable to `upgradetimeoutpernode` since we could have other timeouts in future like wipe timeout.